packages/eslint-plugin-khan/demo/src/foo.tsx

```diff
@@ -1,5 +1,11 @@
 import * as React from "react";
 
+function addStyle(str: string) {
+    // Placeholder example for calling addStyle for aphrodite
+}
+
+const div = addStyle("div");
```

Comment:
I updated the demo project to include a linting error intentionally. Looks like it's [working in CI](https://github.com/Khan/wonder-stuff/actions/runs/12267532496/job/34227812456?pr=1091)! Some thoughts/questions:
- I'm not sure why it shows the errors twice in the PR (same with the other errors)
- Should CI be running linting in the demo project since we expect it to fail?
- I followed the steps in the [Demo readme](https://github.com/Khan/wonder-stuff/blob/main/packages/eslint-plugin-khan/demo/README.md) locally and was running into an error. The linting error also wasn't being shown in VSCode. The error:
```
Oops! Something went wrong! :(

ESLint: 8.40.0

TypeError: Failed to load plugin '@khanacademy' declared in '.eslintrc.js': Class extends value undefined is not a constructor or null

```

Reply:
Try updating `eslint` to 8.55.0.

Reply:
Circling back so I remember what we did:
- We updated eslint versions
- We updated the demo project to use the released `eslint-plugin-khan` package. We'll enable the new addStyle rule in the demo project once these changes are released
- We ignored the demo project from lint checks so that CI can pass (it is expected that the demo project has lint errors so we can test that rules are working as expected!)